clang/test/CodeGenCXX/dynamic-cast-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -I%S %s -triple spirv64-unknown-unknown -fsycl-is-device -emit-llvm -fcxx-exceptions -fexceptions -o - | FileCheck %s
+struct A { virtual void f(); };
+struct B : A { };
+
+// CHECK: {{define.*@_Z1fP1A}}
+// CHECK-SAME:  personality ptr @__gxx_personality_v0
+B fail;
```

Comment:
This adds a third copy of the same test, can we just merge them add add three RUN lines with different CHECK prefixes?